CMakeLists.txt

```diff
@@ -6,11 +6,19 @@ project(mapscore
 ####
 # mapscore OpenGL "proper", i.e. the C++ implementation without language bindings
 ####
-file(GLOB_RECURSE mapscore_SRC
-  "shared/public/*.cpp"
-  "shared/src/*.cpp"
-  "android/src/main/cpp/graphics/*.cpp"
-)
+if(APPLE)
```

Comment:
FWIW, I had been planning to reorganize so that the shared code is built without GL and a separate subproject for the GL code. Then this should generally also work on apple, as long as there is an opengl library (e.g. Mesa).